test/functional/test_node_ports.py

```diff
@@ -240,7 +240,7 @@ def test_start_node_with_conflict_bmc_connection_port(self):
             node2 = model.CNode(self.node_info_2)
             node2.init()
             node2.precheck()
-        except ArgsNotCorrect, e:
+        except ArgsNotCorrect as e:
```

Comment:
what's the difference between these two usages?

Reply:
"ArgsNotCorrect, e" is not standard style, which is not support by python 3.